cpp/include/cudf/io/parquet.hpp

```diff
@@ -205,16 +205,36 @@ class parquet_reader_options {
   [[nodiscard]] data_type get_timestamp_type() const { return _timestamp_type; }
 
   /**
-   * @brief Sets names of the columns to be read.
+   * @brief Sets the names of columns to be read from all input sources.
    *
-   * @param col_names Vector of column names
+   * Applies the same list of column names across all sources. Unlike `set_row_groups`,
+   * which allows per-source configuration, `set_columns` applies globally.
+   *
+   * Columns that do not exist in the input files will be ignored silently.
+   * The output table will only include the columns that are actually found.
```

Comment:
We support nested column selection, right? I wonder if it's a good time to describe how to select a nested column here.

Reply:
Good suggestion